draft-ietf-quic-recovery.md

```diff
@@ -538,11 +538,14 @@ MUST ensure that the crypto retransmission timer is set if there is
 unacknowledged crypto data or if the client does not yet have 1-RTT keys.
 If the crypto retransmission timer expires before the client has 1-RTT keys,
 it is possible that the client may not have any crypto data to retransmit.
-However, the client MUST send a new packet, containing only PING or PADDING
-frames if necessary, to allow the server to continue sending data. If
-Handshake keys are available to the client, it MUST send a Handshake packet,
-and otherwise it MUST send an Initial packet in a UDP datagram of at least
-1200 bytes.
+However, the client MUST send a new packet, containing only PADDING frames if
+necessary, to allow the server to continue sending data. If Handshake keys
+are available to the client, it MUST send a Handshake packet, and otherwise
+it MUST send an Initial packet in a UDP datagram of at least 1200 bytes.
+
+Because packets only containing PADDING do not elicit an acknowledgement,
+they may never be acknowledged, but they are removed from bytes in flight
+when the client gets Handshake keys and the Initial keys are discarded.
```

Comment:
Suggested rewording:
"Because the server will eventually discard Initial and Handshake keys, and because packets containing only PADDING frames do not elicit an acknowledgement, it is possible that no acknowledgement is ever received for these additional packets.  They are however eventually removed from the count of bytes in flight when the client discards the corresponding keys (see {{discarding-keys}})."